packages/main/src/StandardListItem.ts

```diff
@@ -16,31 +16,21 @@ import StandardListItemTemplate from "./generated/templates/StandardListItemTemp
  * This is a list item,
  * providing the most common use cases such as <code>text</code>,
  * <code>image</code> and <code>icon</code>.
- *
- * <h3>CSS Shadow Parts</h3>
- *
- * <ui5-link target="_blank" href="https://developer.mozilla.org/en-US/docs/Web/CSS/::part">CSS Shadow Parts</ui5-link> allow developers to style elements inside the Shadow DOM.
- * <br>
- * The <code>ui5-li</code> exposes the following CSS Shadow Parts:
- * <ul>
- * <li>title - Used to style the title of the list item</li>
- * <li>description - Used to style the description of the list item</li>
- * <li>additional-text - Used to style the additionalText of the list item</li>
- * <li>icon - Used to style the icon of the list item</li>
- * <li>native-li - Used to style the main li tag of the list item</li>
- * <li>content - Used to style the content area of the list item</li>
- * <li>detail-button - Used to style the button rendered when the list item is of type detail</li>
- * <li>delete-button - Used to style the button rendered when the list item is in delete mode</li>
- * <li>radio - Used to style the radio button rendered when the list item is in single selection mode</li>
- * <li>checkbox - Used to style the checkbox rendered when the list item is in multiple selection mode</li>
- * </ul>
+
+ * @csspart title - Used to style the title of the list item
+ * @csspart description - Used to style the description of the list item
+ * @csspart additional-text - Used to style the additionalText of the list item
+ * @csspart icon - Used to style the icon of the list item
+ * @csspart native-li - Used to style the main li tag of the list item
+ * @csspart content - Used to style the content area of the list item
+ * @csspart detail-button - Used to style the button rendered when the list item is of type detail
+ * @csspart delete-button - Used to style the button rendered when the list item is in delete mode
+ * @csspart radio - Used to style the radio button rendered when the list item is in single selection mode
+ * @csspart checkbox - Used to style the checkbox rendered when the list item is in multiple selection mode
  *
  * @constructor
- * @author SAP SE
- * @alias sap.ui.webc.main.StandardListItem
- * @extends sap.ui.webc.main.ListItem
```

Comment:
- Make the CustomListItem class to explicitly implements IListItem in TypeScript
- Remove `@type` tag
- Fix default tag of description property